SwiftReflector/SwiftInterfaceReflector/SwiftInterfaceReflector.cs

```diff
@@ -1645,16 +1654,21 @@ string TargetFromModuleFlags ()
 			return flagsValue;
 		}
 
+		static HashSet<string> ModulesThatWeCanSkip = new HashSet<string> () {
+			"XamGlue",
+			"RegisterAccess",
+			"_StringProcessing",
+			"_Concurrenct",
```

Comment:
Is this spelled correctly?

```suggestion
			"_Concurrent",
```